### PR TITLE
log progress of gProfiler initialization

### PR DESCRIPTION
**Pull request by leonid-granulate**
## Description
Added some info logs indicating different stages of gProfiler initialization .

--verbose flag is removed from docker file;
the output was messy with all the debug stuff, if there is a need this flag can be passed explicitly.

## Motivation and Context
As an existing gProfiler user, I would like to understand the progress of the G-profiler initialization/installation.

## How Has This Been Tested?
Docker image was builded and ran configured to send data to the `performance-studio`.
Logs monitored with `docker logs -f gprofiler`.

## Screenshots:
stdout after the change:
```bash
[11:22:47] Running gprofiler (version 0.0.7)...
[11:22:47] Kernel uname release: 4.15.0-58-generic
[11:22:47] Kernel uname version: #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019
[11:22:47] Total CPUs: 1
[11:22:47] Total RAM: 0.96 GB
[11:22:47] Linux distribution: ('Ubuntu', '18.04', 'bionic')
[11:22:47] libc version: ('glibc', b'2.27-3ubuntu1')
[11:22:48] Connection to server successfully established
[11:22:48] gProfiler initialized and ready to start profiling
[11:22:48] Initializing Java profiler (frequency: 10hz, duration: 60s)
[11:22:48] Initializing Python profiler (frequency: 10hz, duration: 60s)
[11:22:48] Initializing system profiler (frequency: 10hz, duration: 60s)
[11:22:48] Running global perf...
[11:22:48] Profiling process 604 (['/usr/bin/python3', '/usr/bin/networkd-dispatcher', '--run-startup-triggers'])
[11:23:48] Finished profiling process 604
[11:23:54] Finished running global perf
[11:23:56] Successfully upload profile to server
```
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the relevant documentation.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by talsaiag — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -38,4 +38,4 @@ RUN python3 setup.py install
 
 STOPSIGNAL SIGINT
 
-ENTRYPOINT [ "python3", "-m", "gprofiler", "-v" ]
```
Comment (Jongy) [inline Dockerfile:41]: The FileHandler still takes all DEBUG logs, so I guess that's alright.

File: gprofiler/main.py [hunk 2]
```diff
@@ -145,6 +145,7 @@ def _snapshot(self):
                 logger.error(f"Error occurred sending profile to server: {e}")
             except RequestException:
                 logger.exception("Error occurred sending profile to server")
+            logger.info("Successfully upload profile to server")
```
Comment (Jongy) [inline gprofiler/main.py:148]:
```suggestion
            logger.info("Successfully uploaded profiling data to the server")
```

File: gprofiler/client.py [hunk 3]
```diff
@@ -39,6 +39,7 @@ def __init__(self, host: str, key: str, service: str, upload_timeout: int, versi
         self._version: str = version
 
         self._init_session(key, service)
+        logger.info("Connection to server successfully established")
```
Comment (Jongy) [inline gprofiler/client.py:42]:
```suggestion
        logger.info("The connection to the server was successfully established")
```

**Review by Jongy — CHANGES_REQUESTED** [review]
.

File: Dockerfile (hunk 1 above)
Comment (leonid-granulate) [reply Dockerfile:41]: I think we can leave the FileHandler on DEBUG level, and leave the option to user to decide on verbose log to stdout.
Right now the verbose is "hardcoded" and there is not option in `docker` to see only logs > INFO

Comment (Jongy) [reply Dockerfile:44]: kk

File: gprofiler/main.py [hunk 4]
```diff
@@ -146,6 +146,7 @@ def _snapshot(self):
                 logger.error(f"Error occurred sending profile to server: {e}")
             except RequestException:
                 logger.exception("Error occurred sending profile to server")
+            logger.info("Successfully uploaded profiling data to the server")
```
Comment (Jongy) [inline gprofiler/main.py:149]: I think this should go to the `else` clause? Here you'll log "successfully" also after failures.

Comment (Jongy) [reply gprofiler/main.py:149]: So either in the `else` clause of the `try`, or after the `submit_profile(...)` call

**Review by Jongy — APPROVED** [review]

**Pull request merged**